MatrixSDK/Crypto/CryptoMachine/MXCryptoMachine.swift

```diff
@@ -76,6 +79,16 @@ class MXCryptoMachine {
         setLogger(logger: self)
     }
     
+    func onKeysUpload(callback: @escaping () -> Void) {
```

Comment:
Should we name this something more descriptive like `setKeyUploadObserver`?

Reply:
Hm, not sure if thats better, because with the observer pattern you typically also need to explictly call `unset` or `remove`, whereas here it is unset internally on first upload. So perhaps I can rename it to `onInitialKeysUpload`